hw5/GameOfWords.java

```diff
@@ -0,0 +1,71 @@
+package org.hillel.hw5;
+
+import java.util.Objects;
+import java.util.Random;
+import java.util.Scanner;
+
+//Second version of Word Game
+public class GameOfWords {
+    //Main Array of vegetables and fruits
+    private final String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado",
+            "broccoli", "carrot", "cherry", "garlic", "grape", "melon", "leak",
+            "kiwi", "mango", "mushroom", "nut", "olive", "pea", "peanut", "pear",
+            "pepper", "pineapple", "pumpkin", "potato"};
+
+    //Word, which user will write
+    private String inputUserWord;
+    private int indexWord = new Random().nextInt(words.length);
```

Comment:
Initialization of 'indexWord' and 'wordToGuess' can be done in initialization blocks.
Please check the link https://www.google.com/search?q=java+initialization+block&oq=java+intialization+&aqs=chrome.2.69i57j0i13i512l8j0i13i30.3831j0j7&sourceid=chrome&ie=UTF-8